Jamulus.pro

```diff
@@ -155,8 +155,10 @@ win32 {
             }
             # Important: Keep those ASIO includes local to this build target in
             # order to avoid poisoning other builds license-wise.
-            HEADERS += src/sound/asio/sound.h
+            HEADERS += src/sound/asio/sound.h \
+                src/sound/midi-win/midi.h
```

Comment:
As the comment says, this is meant to the just the ASIO stuff.  Can the MIDI bits be added separately?  Maybe enable "win-midi" by default if "not jack-on-windows" and allow it to be turned on in future?